project-base/package.json

```diff
@@ -32,21 +38,23 @@
         "grunt-spritesmith": "^6.6.2",
         "grunt-stylelint": "^0.12.0",
         "grunt-webfont": "^1.7.2",
+        "jest": "^25.1.0",
         "jit-grunt": "^0.10.0",
+        "pofile": "^1.1.0",
         "regenerator-runtime": "^0.13.2",
         "stylelint": "^11.1.1",
         "time-grunt": "^1.4.0",
-        "webpack-notifier": "^1.6.0",
-        "pofile": "^1.1.0"
+        "webpack-notifier": "^1.6.0"
     },
     "dependencies": {
         "@shopsys/framework": "9.0.0-dev.1",
         "codemirror": "^5.49.2",
+        "glob": "^7.1.6",
```

Comment:
glob is not just a dev dependency?

Reply:
a690fce572837c5638850736c313a2e3924eed01